routes/routes.js

```diff
@@ -9,4 +9,5 @@ const controller = require('../handlers/controller');
 module.exports = (app) => {
     app.route('/ping').get(controller.ping).post(controller.ping);
     app.route('/:id').get(controller.getProduct);
```

Comment:
Right now when I make a request it says connection refused. Is this because the heroku instance is not live or because my ip is blocked? I'm requesting codes defined in product.test.js.